src/test_sum_of_nth_terms.py

```diff
@@ -0,0 +1,61 @@
+"""Test for series_sum.py"""
+
+import pytest
+from sum_of_nth_terms import series_sum
+
+
+SERIES_PARAMS_TABLE = [
+    [1, "1.00"],
+    [2, "1.25"],
+    [3, "1.39"],
+    [4, "1.49"],
+    [5, "1.57"],
+    [6, "1.63"],
+    [7, "1.68"],
+    [8, "1.73"],
+    [9, "1.77"],
+    [15, "1.94"],
+    [39, "2.26"],
+    [58, "2.40"],
+    [0, "0.00"],
+]
+
+
+@pytest.mark.parametrize("n, result", SERIES_PARAMS_TABLE)
+def test_series_nth(n, result):
+    """Test basic use cases of series_sum()."""
+    assert series_sum(n) == result
+
+
+def test_random_series_nth():
```

Comment:
Cool, nice job working this into a testing function.  It might be even more cool to factor this into a loop and test 100 random numbers!